### PR TITLE
[236] Style Simpleform submit buttons

### LINKED ISSUE #236: Style all Simpleform Submit buttons
A simple SCSS override

### PR DESCRIPTION
**Pull request by spaceraccoon**
Resolves #236 

Simple override file

### CODE REVIEW

**Comment by spaceraccoon** [comment]
@orenyk @esoterik ready for review! Quick one.

File: app/assets/stylesheets/_simpleform_submit_override.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.simple_form input[type='submit'] {
+  @extend .button;
+}
```
Comment (AlanLiu96) [inline app/assets/stylesheets/_simpleform_submit_override.scss]: don't forget ew lines at end of files :P

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -7,4 +7,5 @@
 @import "flashes";
 @import "forms";
 @import "inline_checkboxes_override";
-@import "draws";
+@import "simpleform_submit_override";
+@import "draws";
```
Comment (orenyk) [inline app/assets/stylesheets/application.scss]: same here

**Comment by orenyk** [comment]
Also don't forget to clean up the commit message - looks like you have the leftovers from before you rebased in there.

**Comment by spaceraccoon** [comment]
Should be good to go! @orenyk @esoterik @AlanLiu96

**Pull request merged**